src/mlpack/core/optimizers/saga/sag_update.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file sag_update.hpp
+ * @authors Prabhat Sharma and Marcus Edel
+ *
+ * Vanilla update for SAG.
```

Comment:
What do you think about ``SAG, a stochastic gradient method for optimizing the sum of a finite set
of smooth functions, where the sum is strongly convex.``